### ISSUE TITLE
GraphQL Introspection and schema linking no longer working since last update

### ISSUE THREAD

**Issue by shaughnlgh** [AntonShuvaev/jet-client-support#95]
**Describe the bug**
GraphQL Introspection no longer working and IntelliJ IDEA now reports Internal plugin error on startup:

```
com.intellij.diagnostic.PluginException: Cannot create extension (class=io.jetclient.graphql.GraphQLEditorHeaderComponentProviderImpl) [Plugin: io.jetclient]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:988)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.doCreateInstance(XmlExtensionAdapter.kt:73)
	at com.intellij.openapi.extensions.impl.XmlExtensionAdapter.createInstance(XmlExtensionAdapter.kt:33)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapter(ExtensionPointImpl.kt:395)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.createExtensionInstances(ExtensionPointImpl.kt:353)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensionList(ExtensionPointImpl.kt:214)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensions(ExtensionPointImpl.kt:229)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensions(ExtensionPointName.kt:51)
	at io.jetclient.bB.a(SourceFile:97)
	at io.jetclient.bB.<init>(SourceFile:54)
	at io.jetclient.bu$6.a(SourceFile:67)
	at io.jetclient.bu$6.invoke(SourceFile:67)
```

In the body section of the JetClient plugin, there used to be a dropdown that allowed to select the graphQL schema to validate against. This no longer exists and writing any query is invalidated. Running the query does produce a successful response, but writing the query now requires a manual introspection and looking up fields.

For reference, here is my existing `graphql.config.xml`, where manual introspection through the GraphQL plugin in IntelliJ works fine.

![Screenshot 2024-07-31 at 10 32 10](https://github.com/user-attachments/assets/16371db1-cde3-4219-9f69-0cb11ff2f6aa)

And here is the invalidated query, with the missing dropdown in the JetClient plugin:

![Screenshot 2024-07-31 at 10 35 25](https://github.com/user-attachments/assets/a0ff281f-dcbf-4874-a365-5a7c78d52b73)

**Environment information:**
- OS: MacOS Sonoma 14.4.1 
- IDE version: IntelliJ IDEA 2024.1.4 (Ultimate Edition), Build #IU-241.18034.62, built on June 20, 2024
- plugin version: JetClient - The Ultimate REST Client (2024.3.1-241)

**Comment by AntonShuvaev** [comment]
Thanks for reporting this issue. I will fix it soon.

**Comment by AntonShuvaev** [comment]
Fixed in version 2024.3.2
